src/Trakx.Circle.ApiClient.Tests/Integration/MockCreator.cs

```diff
@@ -143,4 +143,10 @@ public CardCreationRequest CreatCardRequest() => new()
             IdempotencyKey = Guid.NewGuid().ToString(),
             WalletAddress = GetEthereumAddress()
         };
+
+    public SignetBankCreationRequest GetSignetBankCreationRequest => new()
```

Comment:
Please, don't pollute the code with duplicate methods, this is the same code as the one you had above!